dmgmodify/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/mozilla-services/stubattribution/dmglib"
+	"github.com/mozilla-services/stubattribution/dmgmodify/dmgmodify"
+)
+
+func main() {
+	if len(os.Args) != 5 {
+		log.Fatalf("Usage: %s input.dmg output.dmg sentinel replacement\n", os.Args[0])
+	}
+	input, err := dmglib.OpenFile(os.Args[1])
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer input.Close()
+
+	output, err := os.Create(os.Args[2])
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer output.Close()
+
+	dmgObj, err := input.Parse()
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
Shouldn't we (defer) close the `dmgObj` here too?

Reply:
Actually...no. This is a `DMG` object that has no file handles associated with it.